Makefile

```diff
@@ -26,6 +26,10 @@ build: init
 start: init
 	$(YARN) start$(YARN_ACTION_SUFFIX)
 
+.PHONY: start-with-vnext
```

Comment:
Perhaps just `make start-next`?

I can see wanting a similar `build-next`, but we can wait for now.

Reply:
Done, thanks!

Reply:
Just pushed a commit adding `build-next`, please take a look!